test/itkMontageTestHelper.hxx

```diff
@@ -130,7 +130,7 @@ montageTest(const itk::TileConfiguration<Dimension> & stageTiles,
     origin1[d] = stageTiles.AxisSizes[d] > 1 ? 1 : 0; // support montages of size 1 along a dimension
   }
   size_t    origin1linear = stageTiles.nDIndexToLinearIndex(origin1);
-  PointType originAdjustment = stageTiles.Tiles[origin1linear].Position - stageTiles.Tiles[0].Position;
+  PointType originAdjustment = PointType(stageTiles.Tiles[origin1linear].Position - stageTiles.Tiles[0].Position);
```

Comment:
The difference between two positions is a Vector, mathematically, not a Point. Shouldn't "origin adjustment" be declared as Vector? I guess it depends on what is meant by the term "adjustment".

Reply:
I asked myself the same question. `OriginAdjustment` exists in order to be used [here](https://github.com/InsightSoftwareConsortium/ITKMontage/blob/v0.7.3/include/itkTileMontage.hxx#L167). As such it should be a point. Further reason for keeping it as-is is no need for an interface change.

Reply:
> `OriginAdjustment` exists in order to be used [here](https://github.com/InsightSoftwareConsortium/ITKMontage/blob/v0.7.3/include/itkTileMontage.hxx#L167). As such it should be a point.

Really?

    origin[d] += this->m_OriginAdjustment[d] * nDIndex[d];

Should also work fine if m_OriginAdjustment would be a Vector, as far as I can see.


> Further reason for keeping it as-is is no need for an interface change.

OK! No problem to me.